src/CoreTests/Bases/A11yPropertyTests.cs

```diff
@@ -41,5 +44,27 @@ public void ToStringTest()
             kpVal = ke.Properties[PropertyType.UIA_HasKeyboardFocusPropertyId].ToString();
             Assert.AreEqual("False", kpVal);
         }
+
+        [TestMethod()]
+        public void SimpleCustomPropertyTest() {
+            const int testId = 42;
+            dynamic testValue = 1;
+            const string expectedRendering = "1";
+            A11yProperty.RegisterCustomProperty(testId, new IntTypeConverter());
+            A11yProperty kp = new A11yProperty(testId, testValue);
```

Comment:
Suggestion: Drop the leading 'k' from the property name. I'm guessing it's patterned after the `ke` being used in the existing tests. but `ke` is a remnant of an old class name that eventually became the `A11yElement` class.

Reply:
Todo in a follow-up.